superset/utils.py

```diff
@@ -323,6 +322,11 @@ def base_json_conv(obj):
         return str(obj)
     elif isinstance(obj, timedelta):
         return str(obj)
+    elif isinstance(obj, bytes):
+        try:
```

Comment:
Can we get a unit test here? Also don't we want to raise the error vs. swallow by sending `[bytes]` or even put some type of `logger.error()`?